docs/users-and-roles-swagger.yaml

```diff
@@ -204,6 +204,29 @@ paths:
         responses: {}
         httpMethod: POST
         type: aws_proxy
+    get:
+      summary: Get a stored client and its associated customer
+      security:
+        - CognitoUserPool: [ "https://api.nva.unit.no/scopes/backend", "https://api.nva.unit.no/scopes/frontend", "aws.cognito.signin.user.admin" ]
```

Comment:
-ExternalUserPool: []

Reply:
ExternalCognitoUserPool:
      type: apiKey
      name: Authorization
      in: header
      x-amazon-apigateway-authtype: cognito_user_pools
      x-amazon-apigateway-authorizer:
        type: cognito_user_pools
        providerARNs:
          - Ref: ExternalCognitoAuthorizerArn